modules/observability/proc-admin-enabling-metrics-ocp-operator.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * assemblies/assembly-rhdh-observability.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="proc-admin-enabling-metrics-ocp-operator_{context}"]
+= Enabling metrics monitoring in a {product} Operator installation on an {ocp-short} cluster
+
+You can enable and view metrics for an Operator-installed {product} instance from the *Developer* perspective of the {ocp-short} web console.
+
+.Prerequisites
+
+* Your {ocp-short} cluster has link:https://docs.openshift.com/container-platform/latest/observability/monitoring/enabling-monitoring-for-user-defined-projects.html[monitoring for user-defined projects] enabled.
+
+* You have installed {product} on {ocp-short} using the {product} Operator.
+
+* You have installed the {ocp-short} `oc` CLI.
+
+.Procedure
+
+Currently, the {product} Operator does not support creating a `ServiceMonitor` custom resource (CR) by default. You must complete the following steps to create a `ServiceMonitor` custom resource (CR) to scrape metrics from the endpoint.
+
+. Substitute the values for your environment and then run the following commands to create the `ServiceMonitor` CR:
++
+[source,terminal]
+----
+<custom_resource_name>=my-rhdh
+----
++
+[source,terminal]
+----
+<project_name>=my-project
+----
++
+[source,yaml]
+----
+cat <<EOF > /tmp/$<custom_resource_name>.ServiceMonitor.yaml
```

Comment:
> cat <<EOF > /tmp/$<custom_resource_name>.ServiceMonitor.yaml

The goal with the upstream doc was to provide a way for people to just replace the value of `CR_NAME` and `MY_PROJECT`, then execute the `cat <<EOF ...` command as is right away from their terminal. The shell would then automatically substitute `${CR_NAME}` and `${MY_PROJECT}` accordingly.

Here, it looks like we are expecting users to replace all occurrences of those variables. So IMO, the `cat` command is not required and we can just rewrite this procedure like so:
1. manually create a YAML file named `developer-hub.ServiceMonitor.yaml` (with the content being https://github.com/abrennan89/red-hat-developers-documentation-rhdh/blob/RHIDP-1582/modules/observability/proc-admin-enabling-metrics-ocp-operator.adoc?plain=1#L38-L55, and the variables substituted)
2. Run the `oc apply -f developer-hub.ServiceMonitor.yaml` command

WDYT?